src/test/article.spec.js

```diff
@@ -416,19 +416,59 @@ describe('GET /api/v1/articles/:articleId', () => {
   });
 
   it('should successfully get a specific article', async () => {
-    const validArticle = await BaseRepository.create(db.Article, articleSample);
-    const article = await BaseRepository.findOne(db.Article, validArticle.id);
+    const firstUser = await createUser();
+    const secondUser = await createUser();
+    const thirdUser = await createUser();
+    const fourthUser = await createUser();
 
-    const response = await server().get(`/api/v1/articles/${validArticle.id}`);
-    expect(response.status).to.equal(200);
-    expect(response.body.data.title).to.equal(article.title);
-    expect(response.body.data.description).to.equal(article.description);
-    expect(response.body.data.body).to.equal(article.body);
-    expect(response.body.data.image).to.equal(article.image);
-    expect(response.body.data.slug).to.equal(article.slug);
-    expect(response.body.data.slug).to.be.a('string');
-    expect(response.body.data.authorId).to.equal(article.authorId);
-    expect(response.body.data.publishedDate).to.equal(null);
+    const firstArticle = await createArticle(
+      await generateArticle({
+        authorId: firstUser.id,
+        publishedDate: new Date()
+      })
+    );
+
+    await createArticle(await generateArticle({ authorId: firstUser.id }));
+
+    await rateArticle({
+      articleId: firstArticle.id,
+      userId: secondUser.id,
+      ratings: 5
+    });
+
+    await rateArticle({
+      articleId: firstArticle.id,
+      userId: thirdUser.id,
+      ratings: 3
+    });
+
+    await rateArticle({
+      articleId: firstArticle.id,
+      userId: fourthUser.id,
+      ratings: 3
+    });
+
+    const totalRatings = await BaseRepository.findAndCountAll(db.Rating, {
+      where: { articleId: firstArticle.id }
+    });
+
+    const res = await server().get(`${ARTICLES_API}/${firstArticle.id}`);
+    expect(res.status).to.equal(200);
+    expect(res.body.data[0].id).to.equal(firstArticle.id);
+    expect(res.body.data[0].title).to.equal(firstArticle.title);
+    expect(res.body.data[0].description).to.equal(firstArticle.description);
+    expect(Number(res.body.data[0].count_rating)).to.equal(totalRatings.count);
```

Comment:
Nice work @tolumide-ng with this unit test 👍 
Please assert for the body of the value of the body, slug, image and other important property of the specific article fetched.
This will give more confidence that a valid article was fetched.

Reply:
> Nice work @tolumide-ng 👍
> 
> I'd like you to update the PR description with screenshots of the output on Postman to give confidence that the implementation here works. A look at the link below will help:
> https://github.com/andela/bestpractices/wiki/Git-naming-conventions-and-best-practices

This is not possible at @tobechukwuobitube as this requirement is optional

Reply:
> Nice work @tolumide-ng with this unit test 👍
> Please assert for the body of the value of the body, slug, image and other important property of the specific article fetched.
> This will give more confidence that a valid article was fetched.

Thank you, I have implemented the changes. However, `slugs` and `body` cannot be implemented on this endpoint because it is not relevant to this endpoint as it is not one of outputs. I have however included the image.